Dockerfile

```diff
@@ -0,0 +1,98 @@
+# Prior to building the container, build spark with this command:
+
+# https://spark.apache.org/docs/latest/building-spark.html#building-a-runnable-distribution
+# export MAVEN_OPTS="-Xmx2g -XX:ReservedCodeCacheSize=512m"
+# dev/make-distribution.sh --name custom-spark -Pdeb -U -Phadoop-2.7 -Dhadoop.version=2.8.2 -Pkinesis-asl -Pyarn -Phive -Phive-thriftserver -Dpyspark -Dsparkr -DskipTests -Dgpg.skip=true -Dmaven.javadoc.skip=true package
```

Comment:
This is in the Jenkinsfile too... hadoop 2.7 in one place but 2.8.2 in another?

Reply:
It seems like all the hadoop 2.7 profile does is set the hadoop.version variable. So maybe we don't need -Phadoop-2.7 and just set the hadoop.version. Maybe @markhamstra the maven guru will know